runtime/src/accounts_db.rs

```diff
@@ -4853,22 +4853,13 @@ impl AccountsDb {
         slot: Slot,
         should_flush_f: Option<&mut impl FnMut(&Pubkey, &AccountSharedData) -> bool>,
     ) -> Option<FlushStats> {
-        let is_being_purged = {
-            let mut slots_under_contention = self
-                .remove_unrooted_slots_synchronization
-                .slots_under_contention
-                .lock()
-                .unwrap();
-            // If we're purging this slot, don't flush it here
-            if slots_under_contention.contains(&slot) {
-                true
-            } else {
-                slots_under_contention.insert(slot);
-                false
-            }
-        };
-
-        if !is_being_purged {
+        if self
```

Comment:
I suggest either a comment about this result means `!is_being_purged` or a local of `is_being_purged` like existed before and checking if ! again. I don't know enough from the name of the struct `slots_under_contention` to associate `!is_being_purged`

Reply:
Yeah. Added the comments.